playbooks/roles/ocp-upgrade/tasks/main.yaml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Upgrade cluster
+  shell: |
+    oc adm upgrade --force --to-image="{{ upgrade_image }}" --allow-explicit-upgrade
```

Comment:
Is `force` really needed ?

Reply:
yes as per the command shared by the validation team